src/styles/hc-2018/ternary.styl

```diff
@@ -151,6 +154,7 @@ h4 {
     font-weight: 200;
     border-top: solid var(--secondary-color) 1pt;
     padding-top: 2pt;
+    width: 100%;
```

Comment:
Why is this required?  I thought heading elements were block by default hence would expand to take up the space of their container?

Reply:
Yes, but the border at the top needs to fill the entire container

Reply:
By default the border will fill the entire container since it's block, at least in Safari and Chrome.  Disabling this rule in the web inspector has no effect on the social header on the splash page — is there a browser where this doesn't work?

Reply:
Ah I see what the issue is, just need to add an extra container inside some rows.